libs/island-ui/core/src/lib/BreadcrumbsDeprecated/Breadcrumbs.treat.ts

```diff
@@ -0,0 +1,5 @@
+import { style } from 'treat'
+
+export const divider = style({
+  margin: '0 10px',
+})
```

Comment:
You can skip this treat file and use Box instead

Reply:
this file is now deprecated, i don't think i should be spending time improving this 💁

Reply:
yeah true